### DISCUSSION TITLE
UI bug

### DISCUSSION THREAD

**Discussion by ars3nicum** [Budibase/budibase#9906] [Help] [answered]
I have such bug in a dataprovider filters dialog. After some amounts of filters set, adding a new one from drop down menu don't allows to scroll all the fields available, and part of it goes off screen. Both in Chrome and Safari.
<img width="1148" alt="Screenshot 2023-03-07 at 05 55 38" src="https://user-images.githubusercontent.com/90869728/223373064-becbf57e-b379-4089-8db0-bd8d37806e3a.png">

I was able to do it adding some extra padding to that window through Safari Web Inspector.

**Comment by ConorWebb96** [comment, marked as answer]
Hi @ars3nicum,

I've been able to replicate this and I've created an issue for this #9909 it seems like a minor fix, hopefully one of the engineers can get to it soon and it should be fixed in the next release.

Thank you very much for letting us know about this.

**Reply by ars3nicum** [reply]
Thanks!
